app/src/androidTest/java/org/fossasia/openevent/DatabaseTest.java

```diff
@@ -0,0 +1,66 @@
+package org.fossasia.openevent;
+
+import android.database.sqlite.SQLiteDatabase;
+import android.test.AndroidTestCase;
+
+import org.fossasia.openevent.dbutils.DbContract;
+import org.fossasia.openevent.dbutils.DbHelper;
+import org.fossasia.openevent.dbutils.DbSingleton;
+
+/**
+ * Created by MananWason on 17-06-2015.
+ */
+public class DatabaseTest extends AndroidTestCase {
+    public void TestDropDB() {
+        assertTrue(mContext.deleteDatabase(DbContract.DATABASE_NAME));
+    }
+
+    public void TestCreateDB() {
+        DbHelper dbHelper = new DbHelper(mContext);
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
+        assertTrue(db.isOpen());
+        db.close();
+    }
+
+    public void TestInsertData(){
+        DbHelper dbHelper = new DbHelper(mContext);
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
+
```

Comment:
Yeah first I was writing all tests in this class itself but then I made different classes, so I'll just remove this.